api/src/main/resources/public/api/v1/examples/queue_job_list.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "job":"fatjob2",
+    "runs":[
+      {
+        "backOffUntil":"2017-12-29T23:14:46.758Z",
+        "id":"/fatjob2/201712292252589GUpy",
+        "roles":[
+          "*"
+        ],
+        "runSpec":{
+          "artifacts":[
+
+          ],
+          "cmd":"nuke --dry --master local",
+          "cpus":10,
```

Comment:
I am not sure this is right. Maybe we should take the example out of some request made against metronome? Because e.g. here is cpus but we serialize it as cpu, I don't see disk being serialized, also I see docker instead of container

I am talking about this serialization code https://github.com/dcos/metronome/pull/165/files#diff-2cf575f7407491961e9d5fa3e819122eR255

Reply:
it was "cpu' not "cpus" which was incorrect... fixing.     This is also in RunSpec (which is the wrong place to be looking:).    This json is serialized by the JobRunSpec which serializes cpus and disk.       docker is also correct... metronome was built for docker and translates that to container.docker for marathon but has no concept of container at this point.
This is correct json